bazel/README.md

```diff
@@ -187,7 +187,9 @@ Envoy can produce backtraces on demand and from assertions and other fatal
 actions like segfaults. Where supported, stack traces will contain resolved
 symbols, though not include line numbers. On systems where absl::Symbolization is
 not supported, the stack traces written in the log or to stderr contain addresses rather
-than resolved symbols.
+than resolved symbols. If the symbols were resolved, the address is also included at
+the end of the line. It can be used with a debugging tool like `addr2line` to
+get a precise line number if the required debugging info is available.
```

Comment:
Can you add a suggested `bazel --run_under=....` snippet describing how we can get line numbers?

Reply:
`addr2line` isn't a pass through by itself, although I implemented such a functionality into `stack_decode.py`. If you think it'll actually be useful I can modify `stack_decode.py` to work with the new format backtraces rather than eliminate it. Then you'll be able to use `bazel --run_under=` like before.

Alternatively, I can just write instructions for manually invoking `addr2line` with the right args and the addresses of interest (it just won't be a `--run_under=`).

I have a mild preference for the latter (and let `stack_decode.py` die), WDYT?

Reply:
I feel like run_under is easier to use unless addr2line is destructive to lines that are not part of stack traces. What's the reason you prefer to ask users to do this manually?

Reply:
Yeah, let's revive `stack_decode.py`. Sorry about the futz here, but I love my line numbers! :D

Reply:
No worries! I didn't think anyone was using it! Happy to fix it up to get the best of both worlds.

Reply:
The flow is slightly different. Previously, `stack_decode.py` autodetected the object file name from the log, because backward had this information and we could log it. But we don't have it anymore so the executable file needs to be passed in all circumstances so the usage details of `stack_decode.py` changed slightly. I'm going to update the doc now that mac CI is confirmed to be passing.